src/Compute/Compute/ChangeLog.md

```diff
@@ -23,6 +23,7 @@
 * Added new parameters `ZonePlacementPolicy`, `IncludeZone`, `ExcludeZone`, and `AlignRegionalDisksToVMZone` to cmdlets `New-AzVM` and `New-AzVmConfig`
 * Added new parameter `AlignRegionalDisksToVMZone` to cmdlet `Update-AzVM`.
 * VM/VMSS related cmdlets will now use 2024-11-01 ComputeRP API calls.
+* Added new cmdlets `Add-AzVmssSkuProfileVMSize` and `Remove-AzVmssSkuProfileVMSize` to add and remove VM sizes to and from the VMSS SkuProfile.
```

Comment:
Please always add the new changelog entry at the top (right after **Upcoming Release**). This helps avoid potential merging issues when merging the release branch back into the main branch.